### PR TITLE
chore: not every PR has a related issue, comment the `closes #`

### PR DESCRIPTION
**Pull request by UncleBill**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,2 +1 @@
-<!-- Please refer to the related issue number -->
-closes #
+<!-- Please refer to the related issue number with closes #<issue-d> -->
```
Comment (UncleBill) [inline .github/pull_request_template.md:1]:
```suggestion
<!-- Please refer to the related issue with closes #<issue-d> -->
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,2 +1 @@
-<!-- Please refer to the related issue number -->
-closes #
+<!-- Please refer to the related issue with closes #<issue-d> -->
```
Comment (UncleBill) [inline .github/pull_request_template.md:1]:
```suggestion
<!-- Please refer to the related issue with closes #<issue-id> -->
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,2 +1 @@
-<!-- Please refer to the related issue number -->
-closes #
+<!-- Please refer to the related issue with closes #<issue-id> -->
```
Comment (guanbinrui) [inline .github/pull_request_template.md:1]:
```suggestion
<!-- Please refer to the related issue number -->
<!-- closes # -->
```

Comment (UncleBill) [reply .github/pull_request_template.md:1]: changed `issue number` to `issue id`

**Review by Jack-Works — APPROVED** [review]

**Comment by Jack-Works** [comment]
Actually I think it's ok to leave the `close #`

**Review by septs — CHANGES_REQUESTED** [review]
why?

**Comment by UncleBill** [comment]
> why?

As the title says, not every PR has a related issue.

Well, I'm OK to it, and will close this.

**Pull request closed without merging**